utopiaReservation/buildspec.yml

```diff
@@ -0,0 +1,18 @@
+version: 0.2
+
+phases:
+  install:
+    runtime-versions:
+      nodejs: 8.10
+    commands:
+    - cd utopiaReservation
+    - echo im inside utopia
+    - npm install --silent --no-progress -g npm
+    - npm install --silent --no-progress -g serverless
```

Comment:
you can remove the echo statements since you have already validated that this works